.settings/.gitignore

```diff
@@ -0,0 +1,2 @@
+/org.eclipse.core.resources.prefs
```

Comment:
Can you please remove all of these `.gitignore` files? If you need some additional files ignored, please put them in the top level `.gitignore` file.